docs/modules/commons/pages/variables.adoc

```diff
@@ -65,13 +65,13 @@ NOTE: The scopes names are case-insensitive.
 |Variables are available in the next batches. `next_batches`-scoped variable is not available in batch it's declared within.
 
 |`batch`
-a|Such variables are declared only with `bdd.variables.batch-<batch-number>.` prefix and available throughout the execution of the batch in the suite. An attempt to create the batch-scoped variables during test execution leads to an error.
+a|Such variables are declared only with `variables.batch-<batch-number>.` prefix and available throughout the execution of the batch in the suite. An attempt to create the batch-scoped variables during test execution leads to an error.
```

Comment:
logically it should be `batch-<batch-number>.variables.`, but I would say it's under discussion

Reply:
Really good point, I don't have objections!

Reply:
also in this case: `variables.global.` -> `variables.` (full sync between suite and batch levels)